artiq/gui/entries.py

```diff
@@ -99,9 +99,17 @@ def state_to_value(state):
     def default_state(procdesc):
         if "default" in procdesc:
             return procdesc["default"]
-        elif "min" in procdesc and procdesc["min"] is not None:
-            return procdesc["min"]
         else:
+            have_max = "max" in procdesc and procdesc["max"] is not None
+            have_min = "min" in procdesc and procdesc["min"] is not None
```

Comment:
Are both of these situations (``None`` value and key not present) really legitimately occurring? Why?